package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "teeny-cli",
-    "version": "0.1.0-beta",
+    "version": "0.2.0",
```

I'd still call it beta at this stage
```suggestion
    "version": "0.2.0-beta",
```